lib/ui.dart

```diff
@@ -68,4 +55,16 @@ class AppComponent {
 
     driver(client, users, process);
   }
+
+  addUser(userName) {
+    users = []..addAll(users)..add(userName);
```

Comment:
Why are you reconstructing the list from scratch every time? Does the `add()` not trigger Angular?